server/pkg/schema/schema.go

```diff
@@ -0,0 +1,74 @@
+package schema
+
+import (
+	"github.com/reearth/reearth-cms/server/pkg/id"
+	"github.com/samber/lo"
+	"golang.org/x/exp/slices"
+)
+
+type Schema struct {
+	id        ID
+	workspace id.WorkspaceID
+	fields    []*Field
+}
+
+func (s *Schema) ID() ID {
+	return s.id
+}
+
+func (s *Schema) Workspace() id.WorkspaceID {
+	return s.workspace
+}
+
+func (s *Schema) SetWorkspace(workspace id.WorkspaceID) {
+	s.workspace = workspace
+}
+
+func (s *Schema) HasField(f FieldID) bool {
+	if s == nil {
+		return false
+	}
+	_, found := lo.Find(s.fields, func(g *Field) bool { return g.ID().Equal(f) })
+	return found
```

Comment:
```suggestion
	return lo.SomeBy(s.fields, func(g *Field) bool { return g.ID().Equal(f) })
```

https://github.com/samber/lo#someby is more suitable

Reply:
sure I could not find it before, because I had an old version of `lo`

Reply:
ok to update lo